templates/support.rackspace.com/home.html

```diff
@@ -1,5 +1,12 @@
 {% extends "_layouts/base.html" %}
 
+
+## ALEX DOESNT KNOW WHAT SHE IS DOING HERE
```

Comment:
Is there any particular reason we keep the *header* alone in a separate repo? This doesn't seem sustainable.